src/docs/framework-integration/angular.md

```diff
@@ -204,38 +211,84 @@ Developers can now directly leverage your components in their template and take
 
 ## API
 
-The angularOutputTarget method accepts 5 parameters:
-
 ### componentCorePackage
 
-The title of the Stencil package where components are available for consumers. This is used during compilation to write the correct imports
-for components e.g.
+**Required**
+
+**Type: `string`**
+
+The title of the Stencil package where components are available for consumers (i.e. the `name` property value in your Stencil project's `package.json`).
+This is used during compilation to write the correct imports for components.
+
+For the Ionic Framework, a generated import would look like:
```

Comment:
I like the idea of an example here, but  I think this one assumes some knowledge of how Framework is structured (it assumes some knowledge of `core`, where to find it's directory in the repo and the `package.json` file in that directory).

It may be more beneficial to show a snippet from `npm init stencil component my-component-lib`'s `package.json` (the part that includes the name) and how _that_ name gets translated into an import statement.